securedrop/journalist_app/api.py

```diff
@@ -115,12 +85,10 @@ def get_token() -> Tuple[flask.Response, int]:
 
         try:
             journalist = Journalist.login(username, passphrase, one_time_code)
-            token_expiry = datetime.now(timezone.utc) + timedelta(
-                seconds=TOKEN_EXPIRATION_MINS * 60)
 
             response = jsonify({
-                'token': journalist.generate_api_token(expiration=TOKEN_EXPIRATION_MINS * 60),
-                'expiration': token_expiry,
+                'token': session.get_token(),
+                'expiration': session.get_lifetime(),
```

Comment:
How do we tell the client the new renew mechanism? Which expiration do we want to present here?